decidim-core/app/controllers/concerns/decidim/participatory_space_context.rb

```diff
@@ -63,11 +63,12 @@ def layout
 
     # Method for current user can visit the space (assembly or proces)
     def current_user_can_visit_space?
-      (current_participatory_space.try(:private_space?) &&
-       current_participatory_space.users.include?(current_user)) ||
+      current_user&.admin ||
```

Comment:
Shouldn't this method be handled via permissions? 😕 /cc @isaacmg410 (pinging you since `git blame` points to you)

Reply:
It could be an improvement, yes. 
Also, this method exists before permissons was applied. Right now I don't know the implications that this change can cause. 
In my opinion, this is out of the scope of this PR, because needs a whole refactor that affects, proccesses, assemblies, conferences, and other space.

Reply:
But if you modify the method then we're making the problem worse. I think this PR is a nice opportunity to fix this so it uses the permissions system 😄 

Won't block this PR, just stating my opinion here.